.github/workflows/update-helm-repo.yaml

```diff
@@ -0,0 +1,185 @@
+          helm repo add bitnami https://charts.bitnami.com/bitnami
+          helm repo add hashicorp https://helm.releases.hashicorp.com
+          helm repo add minio https://helm.min.io
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.1.0
+
+      - name: Parse Chart.yaml
+        id: parse-chart
+        run: |
+          cd source
+          changed=$(ct list-changed --config "${CT_CONFIGFILE}")
+          description=$(yq ".description" < ${changed}/Chart.yaml)
+          name=$(yq ".name" < ${changed}/Chart.yaml)
+          version=$(yq ".version" < ${changed}/Chart.yaml)
+          echo "::set-output name=chartpath::${changed}"
+          echo "::set-output name=desc::${description}"
+          echo "::set-output name=tagname::${name}-${version}"
+
+      - name: Install CR tool
+        run: |
+          mkdir "${CR_TOOL_PATH}"
+          mkdir "${CR_PACKAGE_PATH}"
+          mkdir "${CR_INDEX_PATH}"
+          curl -sSLo cr.tar.gz "https://github.com/helm/chart-releaser/releases/download/v1.4.0/chart-releaser_1.4.0_linux_amd64.tar.gz"
+          tar -xzf cr.tar.gz -C "${CR_TOOL_PATH}"
+          rm -f cr.tar.gz
+
+      - name: Create helm package
+        run: |
+          cd source
+          "${CR_TOOL_PATH}/cr" package "${{ steps.parse-chart.outputs.chartpath }}" --config "${CR_CONFIGFILE}" --package-path "${CR_PACKAGE_PATH}"
+          echo "Result of chart package:"
+          ls -l "${CR_PACKAGE_PATH}"
+
+      - name: Create tag and check if exists on origin
+        run: |
+          cd source
+          echo "Making tag ${{ steps.parse-chart.outputs.tagname }}"
+          git tag "${{ steps.parse-chart.outputs.tagname }}"
+
+      - name: Make github release
+        uses: softprops/action-gh-release@v1
+        with:
+          body: |
+            ${{ steps.parse-chart.outputs.desc }}
+
+            Source commit: https://github.com/${{ github.repository }}/commit/${{ github.sha }}
+
+            Tag on source: https://github.com/${{ github.repository }}/releases/tag/${{ steps.parse-chart.outputs.tagname }}
+          files: |
+            ${{ env.CR_PACKAGE_PATH }}/${{ steps.parse-chart.outputs.tagname }}.tgz
+            ${{ env.CR_PACKAGE_PATH }}/${{ steps.parse-chart.outputs.tagname }}.tgz.prov
+          repository: grafana/helm-charts
+          tag_name: ${{ steps.parse-chart.outputs.tagname }}
+          token: ${{ secrets.helm_repo_token }}
+
+      - name: Push release tag on origin
+        run: |
+          cd source
+          echo "Pushing tag ${{ steps.parse-chart.outputs.tagname }}"
+          git push origin "${{ steps.parse-chart.outputs.tagname }}"
+
+      - name: Update helm repo index.yaml
+        run: |
+          cd helm-charts
+          "${CR_TOOL_PATH}/cr" index --config "${CR_CONFIGFILE}" --token "${{ secrets.helm_repo_token }}" --index-path "${CR_INDEX_PATH}" --package-path "${CR_PACKAGE_PATH}" --pr
```

Comment:
Just out of curiosity, is there a reason not to use the chart-releaser-action that we use elsewhere in this repo?

https://github.com/grafana/helm-charts/blob/main/.github/workflows/release.yaml#L48

Reply:
@krajorama

Reply:
I've tried to run it in the mimir repo, but didn't work as it created the release in mimir repo and tried to update its gh-pages (which would put the index.yaml in the wrong place).

On the other hand, I'm realizing now that I ended up checking out the helm-charts repo anyway, so potentially we could copy the code from mimir to the new repo, stage the files (for [git diff](https://github.com/helm/chart-releaser-action/blob/a3454e46a6f5ac4811069a381e646961dda2e1bf/cr.sh#L250) to work) and run the chart releaser.